datalad/__init__.py

```diff
@@ -130,6 +130,11 @@ def setup_package():
 """)
         _TEMP_PATHS_GENERATED.append(new_home)
 
+    # Re-load ConfigManager, since otherwise it won't consider global config
+    # from new $HOME (see gh-4153
+    global cfg
```

Comment:
Why `global`? There is only a method call, no assignment.

Reply:
Just being explicit. Doesn't do any harm, does it?

Reply:
I would also vote to remove it until we need it -- other variable(s) in the function do not use `global`

Reply:
Done.